setup.py

```diff
@@ -44,7 +44,13 @@
         "protobuf",
         "torchvision",
     ],
-    "neuronx": ["neuronx-cc==2.*", "torch-neuronx", "torchvision"],
+    "neuronx": [        
+        "wheel",
+        "neuronx-cc==2.*", 
+        "torch-neuronx", 
```

Comment:
not sure if thats a good idea since if you run pip install you cannot install them since they are not on `pypi`

Reply:
Yeah, you are right. Better we set up aws's repo as an extra path to pip. I just added the `PIP_EXTRA_INDEX_URL` env variable in setup. WDYT?

Reply:
Does `PIP_EXTRA_INDEX_URL` work when publishing to pypi and then installing it?

Reply:
Good question, I just tested with `pip install .[neuronx]`. I can upload to testpypi for a double check.

Reply:
lets test and make sure it uses the correct index. Since AWS published dummy packages which you can still install from the normal pypi

Reply:
@philschmid Just tested, seems working decently:
![image](https://user-images.githubusercontent.com/44135271/236277019-d74ddf24-8b2f-457f-b8b5-379dcdab1330.png)
You can find the test package here: https://test.pypi.org/project/optimum-neuron/

Reply:
Thats awesome!